### ISSUE TITLE
Macarius in Simeon the bishop text?

### ISSUE THREAD

**Issue by nathangibson** [srophe/srophe-app-data#554]
Is http://syriaca.org/person/856 (Macarius) supposed to be marked as commemorated in Simeon the bishop (text) http://wwwb.library.vanderbilt.edu/work/802 ? The person record doesn't cite this work, so I was wondering if it is a mixup.

**Comment by davidamichelson** [comment]
Not all of the commemorations are in yet. By monday I hope

**Comment by jsaintlauren** [comment]
No, that is wrong.  Simeon of Beth Arsham never knew Macarius :)

**Comment by nathangibson** [comment]
@davidamichelson Are you working on commemorations? Sorry I didn't realize. I think I just managed today to get many of the remaining ones in today using the BHS citations in persons. I hope I didn't mess up anything you were doing.

**Comment by wlpotter** [comment]
@jsaintlauren https://syriaca.org/work/802 is listed as commemorating Macarius (https://syriaca.org/person/856), which I believe should be removed? 

Note that we don't have an attestation link from https://syriaca.org/person/856 to https://syriaca.org/work/802, which I believe is correct.
